src/styles/app.css

```
@@ -146,8 +146,47 @@
+		opacity: 0.78;
+		outline: 3px solid #ef4444;
+		outline-offset: 3px;
+		filter: grayscale(0.45) drop-shadow(0 0 1px #dc2626)
+			drop-shadow(0 0 6px #ef444499);
+	}
+
 	.gh-json-viewer pre {
 		margin: 0;
+		white-space: pre-wrap;
+		overflow-wrap: anywhere;
+		word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)